### DISCUSSION TITLE
Use flowbite in a react project without flowbite-react

### DISCUSSION THREAD

**Discussion by johngouf** [themesberg/flowbite#311] [Q&A]
Hi! I was looking for some guidance about how to setup a react project created with create-react-app without using flowbite-react.
Many thanks!

**Comment by MarioStipetic** [comment]
Hi I am not an expert, but please try this:
https://flowbite.com/docs/getting-started/quickstart/#include-via-cdn

Add those two scripts inside the "index.html" file in your react folder.

**Reply by MarioStipetic** [reply]
Actually, the regular Flowbite elements that are linked via CDN seems not to work inside react to me for some reason.
Please let me know if you manage to make it work somehow.

**Comment by zes** [comment]
I noticed a curious behavior : 

First of all, I installed Flowbite with NPM (https://flowbite.com/docs/getting-started/quickstart/#require-via-npm) but I don't add script tag in index.html.
(And for information I test if it works displaying a Popover.)

If I add `import 'flowbite';` in `index.tsx` I never see the popover.
If I add it in `App.tsx` I don't see it after a fresh `yarn start` but if I comment the `import 'flowbite';`, save the file so app is compiling again and uncomment it and re-save the file (so app is compiling again), I can see the popover.
